guides/v2.1/frontend-dev-guide/css-topics/using-icons.md

```diff
@@ -0,0 +1,40 @@
+---
+group: frontend-developer-guide
+title: Using custom icons
+functional_areas:
+  - Frontend
+---
+
+The Magento application directly extends the Magento 2 Blank theme but implements the font icon set from the Luma theme.
+
+The package includes CSS adjustments to scale font sizes in keeping with the Luma icons (which represent a size more typical of available icon fonts). The theme is intended to serve as a base for extending with your own theme and replacing icons with your own.
+
+Using fonts for icons makes sense for a lot of reasons. In reality, iconography has always had more in common with text than content images icons are glyphs that communicate meaning to the audience. And in technical terms, font glyphs are vector information that can scale up to any resolution, The native Magento theme provides the framework you need for implementation.
```

Comment:
Hi @abrarpathan19, this topic is a plagiarism, copying/pasting some text from external sources isn't a good practice and it should be avoided, because the duplicated content has a negative impact on the SEO. 

The same thing with the next paragraph.

Source: https://www.classyllama.com/blog/icon-fonts-magento-2-foundation